pipelines/upgrade_pipeline.yml

```diff
@@ -25,15 +23,15 @@
     - role: forklift_versions
       scenario: "{{ pipeline_type }}"
       scenario_version: "{{ forklift_upgrade_version_start }}"
-    - umask
-    - selinux
-    - etc_hosts
-    - foreman_server_repositories
-    - update_os_packages
-    - haveged
-    - enable_ipv6
-    - disable_firewall
-    - foreman_installer
```

Comment:
wasn't the idea of @ekohl's comment to have `foreman_installer` in both pipelines, so the RPM step is done in parallel on both machines, instead of removing it in both and thus not doing parallel at all?